Automattic/wp-calypso
# Woo Tailored Onboarding: Add new domain/plans steps to ecommerce flow

## This is work in progress!

* This PR builds on top of https://github.com/Automattic/wp-calypso/pull/68722 
* Adds Domains & Plan steps to the new `ecommerce` flow

#### Testing Instructions
* Make sure the flag `signup/tailored-ecommerce` is enabled in your environment
* Access `/setup?flow=ecommerce` and you'll be redirected to `/setup/domains?flow=ecommerce`
* Pick a domain
![jBqZvh.png](https://user-images.githubusercontent.com/3801502/199599045-9f548ac2-fc6c-47de-8114-d08ac694504e.png)
* You'll be redirected to `/setup/plans?flow=ecommerce`
* Pick a Plan
![4XM6qq.png](https://user-images.githubusercontent.com/3801502/199599241-bea6b4c3-0944-4cf8-9a11-5ca9ecc7e04f.png)
* If you picked a paid domain/plan you'll go to checkout
![fRQXUo.png](https://user-images.githubusercontent.com/3801502/199599545-785e4cc9-e768-4655-9a16-20770a943b46.png)
* After checkout, you'll be redirected to the `intro` step and your site will already be created
![Kzuj86.png](https://user-images.githubusercontent.com/3801502/199599954-b3782d86-09e7-476d-8438-1deeb554b45a.png)
* Click create your store and you'll be redirected to `storeProfiler`
![hi6n1t.png](https://user-images.githubusercontent.com/3801502/199600105-22c535f9-40d2-4722-9125-1ab581bbdcf9.png)
* Click Continue and you'll be redirected to `designCarousel` step
![zjAGwD.png](https://user-images.githubusercontent.com/3801502/199600270-b58849e5-e9de-4f1f-91f0-9c1800953a02.png)
* Choose a theme and click Continue.
* You should be redirected to `/home/<new-site-slug>`. 
**The Site should have the plan/domain/theme you chose in the flow.**
